pyproject.toml

```diff
@@ -100,77 +100,55 @@ line-length = 88
 
 [tool.ruff]
 line-length = 88
+target-version = "py39"
+extend-exclude = ["doc/*", "benchmarks/*", "versioneer.py", "geopandas/_version.py"]
+
+[tool.ruff.lint]
 select = [
     # pyflakes
     "F",
     # pycodestyle
     "E",
     "W",
-    # flake8-2020
-    "YTT",
```

Comment:
Removed because this just contains some rules about accessing `sys.version` elements, which we currently don't really do anywhere